src/utils/formatBalance.ts

```diff
@@ -64,7 +64,12 @@ export const formatFixedNumber = (number: FixedNumber, displayDecimals = 18, dec
 
 export const formatLocalisedCompactNumber = (number: number): string => {
   const codeFromStorage = getLanguageCodeFromLS()
-  return new Intl.NumberFormat(codeFromStorage, {
+
+  const isClient = typeof window === 'object'
+
+  if (!isClient || !window?.Intl) return ''
```

Comment:
@chefilip Is this for SSR? Is there some bug currently regarding this?
I wonder if we should install / write some polyfil instead of returning `''`

Reply:
Oh. It's on client-side. Some browsers don't support it.

Reply:
I think we have to handle Intl not supported browser. I don't wan to add polyfill here just for 10 users.
can we just return the `codeFromStorage` when isClient and no Intl support?

Reply:
Done

Reply:
sorry, should be `number` instead 😅 
Intl should works on node js.. umm
<img width="387" alt="CleanShot 2022-03-01 at 19 20 41@2x" src="https://user-images.githubusercontent.com/94336009/156160191-3fb6c40d-c74b-4f23-b447-98fb213be829.png">

Reply:
Done again. :)